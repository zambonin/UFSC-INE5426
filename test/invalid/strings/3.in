# assignment of a character array to a character
char a = "s"
