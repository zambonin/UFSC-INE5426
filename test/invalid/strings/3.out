[Line 2] semantic error: attribution operation expected character but received character array
